reframe/core/meta.py

```diff
@@ -8,12 +8,37 @@
 #
 
 from reframe.core.exceptions import ReframeSyntaxError
+import reframe.core.directives as ReframeDirectives
```

Comment:
I moved the parameter logic to `reframe/core/parameters.py`. The reason why I've gone for parameters is because all the other files in `reframe/core` are also nouns.